BRPeerManager.c

```diff
@@ -1789,9 +1761,12 @@ void BRPeerManagerConnect(BRPeerManager *manager)
 
         if (array_count(manager->peers) < manager->maxConnectCount ||
             manager->peers[manager->maxConnectCount - 1].timestamp + 3*24*60*60 < now) {
-            //TODO: WIP here add logic enable/disable
-//            _BRPeerManagerFindPeers(manager);
-            _BRPeerManagerFindPeersV2(manager);
+            peer_log(&BR_PEER_NONE, "isFeatureSelectedPeersOn: %d", (manager->isFeatureSelectedPeersOn ? manager->isFeatureSelectedPeersOn(manager->info) : 0));
+            if (manager->isFeatureSelectedPeersOn) {
+                _BRPeerManagerFindPeersV2(manager);
+            } else {
+                _BRPeerManagerFindPeers(manager);
+            }
         }
 
         array_new(peers, 100);
```

Comment:
@andhikayuana ...I was mistaken and this list of viable nodes may actually be ~150. So we should define this array to `175`

Reply:
```
 array_new(peers, 100);
```
you mean this part need to be increased to 175 right?